cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -407,15 +407,33 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         validateDueIn: function() {
             if (this.getValue() > 18){
                 this.$('#due-num-weeks-warning-max').show();
+                this.$('#projected').hide();
                 BaseModal.prototype.disableActionButton.call(this.parent, 'save');
             }
             else if (this.getValue() < 1){
                 this.$('#due-num-weeks-warning-min').show()
+                this.$('#projected').hide();
                 BaseModal.prototype.disableActionButton.call(this.parent, 'save');
             }
+            else if (!this.getValue()){
+                this.$('#due-num-weeks-warning-max').hide();
+                this.$('#due-num-weeks-warning-min').hide();
+                this.$('#projected').hide();
+            }
             else {
                 this.$('#due-num-weeks-warning-max').hide();
                 this.$('#due-num-weeks-warning-min').hide();
+                this.$('#projected').hide();
+                if (this.model.get('start')){
+                    var startDate = DateUtils.parseDateFromString(this.model.get('start'))
+                    this.$("#startDate").html(startDate.toDateString()); // xss-lint: disable=javascript-jquery-html
```

Comment:
why are these necessary xss-lint disables necessary? i always get a bit cautious when i see xss-lint stuff getting disabled in case there is a potential security issue

Reply:
yeah I got a bunch of `javascript-jquery-html` errors in the quality check, so disabling was the solution for it. But if you know better solutions to this please let me know!

Reply:
fixed!